apps/aws-notebook/start_jupyter_notebook.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+
+ECS_INSTANCE_STATE="/run/sandbox/fs/resources/aws-ecs-service/state"
+JUPYTER_KERNEL_DIR="/usr/local/share/jupyter/kernels/ECS IPython Kernel"
+JUPYTER_KERNEL_FILE=$JUPYTER_KERNEL_DIR"/kernel.json"
+IP_PORT=22
+
+function fatal() {
+  echo "$@" >&2
+  exit 1
+}
+
+function ecs_ip() {
+    local ip_addr
+    ip_addr=`jq .task_private_ip.value $ECS_INSTANCE_STATE`
+    if [ $? -eq 0 ] && [ ! -z "$ip_addr" ]; then
+        ip_addr=`echo $ip_addr | sed 's/"//g'`
+        echo "$ip_addr"
+    fi
```

Comment:
Please check my previous comment, this is too complicated:

```sh
local ip_addr="$(jq -cMr .task_private_ip.value $ECS_INSTANCE_STATE 2>/dev/null || true)"
[[ "$ip_addr" == "null" ]] || echo "$ip_addr"
```

That's it. Please read the doc to understand `-cMr` for jq.